src/clojure/clj_yaml/core.clj

```diff
@@ -157,3 +157,23 @@
                             :allow-recursive-keys allow-recursive-keys
                             :allow-duplicate-keys allow-duplicate-keys)
                  string) keywords))
+
+;; From https://github.com/metosin/muuntaja/pull/94/files
+(defn generate-stream
+  "Dump the content of data as yaml into writer."
+  [writer data & opts]
+  (.dump ^Yaml (apply make-yaml opts) (encode data) writer))
+
+(defn parse-stream
+  [^java.io.Reader reader & {:keys [keywords eof] :or {keywords true} :as opts}]
+  (letfn [(parse []
+            (decode (.load ^Yaml (apply make-yaml (flatten (seq opts)))
```

Comment:
Flatten is usually a bit of an anti-pattern. If the goal is to go from a map to a seq of keys and values, you can e.g. do:

```
user=> (into [] cat {:a 1 :b 2})
[:a 1 :b 2]
```

Reply:
:pray: